frontend/src/app/shared/shared.module.ts

```diff
@@ -36,7 +37,8 @@ import { UserChipList } from './user-chip-list/user-chip-list.widget';
     EventCard,
     EventList,
     UserLookup,
-    UserChipList
+    UserChipList,
+    SlackInviteBox
```

Comment:
Nit: I might argue that `SlackInviteBox` can be moved to the App Module, since it is only used by one component! In the future though, we probably need to move this into its own module.

Reply:
I concur, may as well go ahead and do this move ahead of initial merge. I'll take a stab.